Updated Cloud Minor Upgrades article (#6537)
umbraco/UmbracoDocs: umbraco-cloud/product-upgrades/minor-upgrades.md

---
description: This article explains how Minor upgrades work in Umbraco Cloud.
---

# Minor Upgrades

When a new minor version is released, there are two ways to upgrade your project:

- [Automatic Minor Upgrades](#automatic-minor-upgrades)
- [Manual Minor Upgrades](#manual-minor-upgrades)

## Automatic Minor Upgrades

To enable automatic minor upgrades, follow these steps:

1. Go to your Umbraco Cloud project.
2. Navigate to **Configuration** -> **Automatic Upgrades**.

    <figure><img src="../.gitbook/assets/image (47).png" alt="Settings Umbraco Cloud"><figcaption><p>Settings Umbraco Cloud</p></figcaption></figure>
3. Enable **Automatic Minor Upgrades**.

    <figure><img src="../.gitbook/assets/image (10) (1).png" alt=""><figcaption><p>Enable Minor Upgrades</p></figcaption></figure>

> With automatic upgrades enabled, it will be applied to all products on Umbraco Cloud: Umbraco CMS, Umbraco Forms, and Umbraco Deploy. Your project does not need to be running the latest minor version for automatic upgrades to work. The project will be upgraded to the latest minor version when it is released.

Reviewer: Was a little confused about the "it will be applied", so attempted a rewrite of this sentence. Let me know what you think!

```suggestion
With automatic upgrades enabled, all products on Umbraco Cloud will automatically be upgraded. This includes Umbraco CMS, Umbraco Forms, and Umbraco Deploy. Your project does not need to be running the latest minor version for automatic upgrades to work. The project will be upgraded to the latest minor version when it is released.
```

> If you create a new project on Umbraco Cloud, by default, automatic minor upgrades will be enabled.

Reviewer: ```suggestion
If you create a new project on Umbraco Cloud automatic upgrades are enabled by default.
```

{% endhint %}

A development environment is included in all Umbraco Cloud plans, except Starter. Find pricing details for Umbraco Cloud Starter plans on our [website](https://umbraco.com/products/umbraco-cloud/pricing).

## Manual Minor Upgrades

A manual upgrade involves a more hands-on approach, where the upgrade process is initiated and controlled by the user or development team. This allows for greater flexibility and oversight, enabling teams to test and adapt the upgrade to their specific needs and configurations. A manual upgrade provides an opportunity to thoroughly test the new version in a controlled environment before applying it to live production environments. This ensures compatibility and minimizes disruptions.

For more information about manual minor upgrades, see the [Manual upgrade of Umbraco CMS](../product-upgrades/manual-upgrades/manual-cms-upgrade.md) article.

## Troubleshooting Automated Minor Upgrades

Umbraco Cloud supports performing minor upgrades to your projects automatically. The feature is available when a new minor version of Umbraco is released (for example 10.5.0 or 10.6.0).

> The upgrade will cover most issues it encounters, but some Umbraco configurations may need some manual intervention. This is usually related to custom code that depends on APIs that have changed or been removed in the new minor version.

Reviewer: ```suggestion
The upgrade will resolve most issues it encounters, but some Umbraco configurations may need manual intervention. This is usually related to custom code that depends on APIs that have changed or been removed in the new minor version.
```
